lib/dialects/abstract/query-generator.js

```diff
@@ -205,6 +211,35 @@ var QueryGenerator = {
         emptyQuery += ' RETURNING *';
       } else if (!!this._dialect.supports.returnValues.output) {
         outputFragment = ' OUTPUT INSERTED.*';
+
+        //To capture output rows when there is a trigger on MSSQL DB
+        if (modelAttributes && options.hasTrigger && this._dialect.name === 'mssql') {
```

Comment:
`this._dialect.supports` - defined in `dialect/index`. Probably `this._dialect.supports.tmpTableTrigger` or something similar